spec/std/io/io_spec.cr

```diff
@@ -333,6 +333,14 @@ describe IO do
       end
     end
 
+    it "#read_string?" do
+      io = SimpleIOMemory.new("hello world")
+      io.read_string?(5).should eq("hello")
+      io.read_string?(1).should eq(" ")
+      io.read_string?(0).should eq("")
```

Comment:
This scenario is not described in the documentation. I know originally isn't described for `#read_string` either, but might be worth adding it.

Thank you.